docs/guide/personalization.md

```diff
@@ -127,9 +127,9 @@ Place this in a settings file which won't be affected by an update to the Recomm
 
 Visitor events (clicks, buys, ...) on the site need to be sent to the Personalization Solution for the recommendations to be calculated. The content types that are marked to be tracked are also exported to the Personalization Engine. Please note that you can only recommend what you track!
 
-By defining the Content Types in the local `app/config/config.yml` file, the content will be initially exported by a script. After this, it will be kept in sync with the Personalization Solution everytime a change occurs in the eZ Platform back office.
+By defining the Content Types in the local `app/config/config.yaml` file, the content will be initially exported by a script. After this, it will be kept in sync with the Personalization Solution everytime a change occurs in the eZ Platform back office.
```

Comment:
I'm not sure about `config.yaml` file, name feels too generic for me. At least file path is invalid - it should be more like`config/packages/config.yaml`.